modules/upgrade/pages/migrate/data-migration.adoc

```diff
@@ -2,9 +2,7 @@
 :description: Migrate external data to Redpanda with MirrorMaker 2.
 :page-aliases: data-management:data-migration.adoc, manage:data-migration.adoc
 
-One of the more elegant aspects of Redpanda is that it is compatible with the xref:develop:kafka-clients.adoc[Kafka API and ecosystem].
-So when you want to migrate data from Kafka or replicate data between Redpanda clusters,
-the MirrorMaker 2 (MM2) tool bundled in the https://kafka.apache.org/downloads[Kafka download package^] is a natural solution.
+When you want to migrate data from Kafka or replicate data between Redpanda clusters, you can use the MirrorMaker 2 (MM2) tool bundled in the https://kafka.apache.org/downloads[Kafka download package^].
```

Comment:
Regarding this suggestion: https://github.com/redpanda-data/docs/pull/58#issuecomment-1747190424 Did you mean, take out the link to Kafka downloads here, and then edit [line 12](https://github.com/redpanda-data/docs/pull/58/files#diff-312d00c897a59bbc918242d8ec5ccaaf2ea580eb5cd30cd81976228719749cbdR12) so it has the downloads link there instead?

Reply:
Yes!